blockchain/wire.go

```diff
@@ -10,4 +12,5 @@ var cdc = amino.NewCodec()
 func init() {
 	RegisterBlockchainMessages(cdc)
 	types.RegisterBlockAmino(cdc)
+	evidence.RegisterEvidenceMessages(cdc)
```

Comment:
we actually want `types.RegisterEvidences`. This is registering the evidence reactor messages, which we don't care about here. We need the evidence data type itself, which is what is included in the blockchain

Reply:
oh what. It looks like types.RegisterBlockAmino already does that. We may need to investigate further